bin/hjs-dev-server.js

```diff
@@ -21,7 +22,21 @@ try {
 }
 
 var serverConfig = config.devServer
+var https = serverConfig.https
 var app = express()
+
+var createServer = require(https ? 'https' : 'http').createServer
+var server
+
+if (https) {
+  server = createServer({
+    key: fs.readFileSync(path.resolve(__dirname, '../resources/hjs-webpack-localhost.key')),
+    cert: fs.readFileSync(path.resolve(__dirname, '../resources/hjs-webpack-localhost.crt'))
+  }, app)
```

Comment:
@dignifiedquire That's a good idea. I'm gonna merge in the interest of quickness for @tdd, then add that and add back the original https docs and release.